std/stdio.d

```diff
@@ -3315,6 +3358,23 @@ unittest
 }
 
 /**
+Forwards to `stdout.print`.
+*/
+void print(string separator = " ", string eol = "\n", S...)(S args)
+{
+    trustedStdout.print!(separator, eol)(args);
+}
+
+///
+unittest
+{
+    void printCSVRow(Data...)(Data data)
+    {
+        print!", "(data);
```

Comment:
Echoing my statement above, the unit test should not print to stdout. The unit tests should either
1. Replace stdout with a temp file (temporarily)
2. Use a temp file directly.

You could also have a unit test that just verified the call could compile. Wouldn't make a good example though.